libs/sql-schema-describer/Cargo.toml

```diff
@@ -9,13 +9,17 @@ failure = "0.1"
 chrono = { version = "0.4" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-quaint = { git = "https://github.com/prisma/quaint.git" }
 url = "1.7.2"
 log = "0.4"
 regex = "1.2"
 async-trait = "0.1.17"
 futures = { package = "futures", version = "0.3", features = ["compat"] }
 
+[dependencies.quaint]
+git = "https://github.com/prisma/quaint.git"
+default-features = false
+features = ["sqlite", "json-1", "postgresql", "uuid-0_7", "chrono-0_4", "mysql"]
```

Comment:
is this still required?

Reply:
There is no negative feature selection. Like select all but not this.